frontend/index.html

```diff
@@ -2,11 +2,11 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/images/logo.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>twp</title>
   </head>
-  <body>
+  <body style="background-color: var(--bg)">
```

Comment:
using variables causes flashing on Firefox w/ Linux w/ `npm run dev` because `index.html` arrives sooner than the css
hardcoding the value is fine if there isn't any alternative solution

Reply:
it cause flash but it's not white flash on firefox and safari in mac.
do yours still be white flash?

Reply:
yes, I'm not sure if it's because of my default system color theme setting or something else